### DISCUSSION TITLE
Konsta Svelte styles not rendering at all

### DISCUSSION THREAD

**Discussion by JoshuaRileyDev** [konstaui/konsta#73] [Q&A]
Hello,

I've got an issue with Konsta Svelte, I've started up a new SvelteKit project 3 times now with TailwindCSS configured properly and Konsta installed via NPM, however the styles for the KonstaUI components aren't rendering but normal HTML with TailwindCSS classes are rendering fine, it's very strange

![CleanShot 2022-08-31 at 16 32 38](https://user-images.githubusercontent.com/59296334/187718789-a9bdfb42-7b58-4f56-bc64-8e8f98a6c2ab.png)

**Comment by daniromome** [comment]
![image](https://github.com/konstaui/konsta/assets/20412235/50fd916d-fcb8-4167-baca-4006985e1a24)
I have this same issue, I'm using the latest version of svelte/kit & konsta with svelte kit's [static adapter](https://kit.svelte.dev/docs/adapter-static)

**Comment by daniromome** [comment]
![image](https://github.com/konstaui/konsta/assets/20412235/a621fc3b-3f98-49b3-88a3-02642f79aa12)

I was missing postcss.config.js file in the root of my project, after adding it styles are loaded properly, you can take a look at the following repo as an example https://github.com/Imtiyaz-cuepra/sveltekit-konstaui-capacitor
